include/clangmetatool/collectors/find_calls_data.h

```diff
@@ -0,0 +1,42 @@
+#ifndef INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_DATA_H
+#define INCLUDED_CLANGMETATOOL_COLLECTORS_FIND_CALLS_DATA_H
+
+#include <clang/Frontend/CompilerInstance.h>
+#include <clang/ASTMatchers/ASTMatchers.h>
+#include <clang/ASTMatchers/ASTMatchFinder.h>
+#include <clang/AST/Expr.h>
+
+namespace clangmetatool {
+  namespace collectors {
+
+  struct FindCallsData {
+    std::map<const clang::FunctionDecl*,
+             const clang::CallExpr*> call_context;
+    std::map<const clang::CallExpr*,
+             const clang::DeclRefExpr*> call_ref;
+    std::map<const clang::CallExpr*,
+             const clang::DeclRefExpr*> call_argref;
+    std::map<const clang::CallExpr*,
+             const clang::StringLiteral*> call_argstr;
```

Comment:
Could we make this 
```
    std::map<
             std::pair<const clang::CallExpr*, int>,
             const clang::StringLiteral* > call_argstr;
```

That way you could just traverse all arguments and assign it to the map with the positional index of that argument... (and the same for `call_argref`